.github/workflows/ci.yml

```diff
@@ -0,0 +1,94 @@
+name: Package and Publish to Visual Studio Marketplace
+
+on:
+  push:
+    branches:
+      - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+
+      - name: Set Git configuration
+        run: |
+            git config user.name "${GITHUB_ACTOR}"
+            git config user.email "${GITHUB_ACTOR}@bots.github.com"
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Bump version and create Git tag
+        uses: Aqovia/npm-version-increment@v1.0.4
+        id: version
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        
+      - name: Extract extensionPack from package.json
+        run: |
+            # Get the "extensionPack" section from package.json and save it in a file
+            node -p "JSON.stringify(require('./package.json').extensionPack, null, 2)" > extensionPack.json
+            # Output the contents of extensionPack.json to the build log
+            #cat extensionPack.json
```

Comment:
Do we need these comments?

Reply:
Not if you don't want them. The first one seems useful to me but I can delete it.